libsol/transaction_summary.c

```diff
@@ -98,22 +105,61 @@ void transaction_summary_reset() {
     explicit_bzero(&G_transaction_summary_text, TEXT_BUFFER_LENGTH);
 }
 
+static uint64_t transaction_summary_get_compute_units_limit() {
+    if (G_transaction_summary.compute_units_limit.u64 == 0) {
+        // Prioritization fee is calculated by multiplying this value with compute unit price,
+        // so it cannot be equal to zero
+        G_transaction_summary.compute_units_limit.u64 = DEFAULT_COMPUTE_UNIT_LIMIT;
+    }
+
+    return G_transaction_summary.compute_units_limit.u64;
+}
+
+uint64_t calculate_additional_transaction_fees(){
```

Comment:
let's just display the priority fee and limit individually rather than try to resolve a final fee like this.  as the compute budget gets more features, this is just going to need constantly updated and become overly complex